### PR TITLE
[Fixtures] adding default_tax_zone to ChannelFixture

### PR DESCRIPTION
**Pull request by ktzouno**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php [hunk 1]
```diff
@@ -123,6 +133,9 @@ protected function configureOptions(OptionsResolver $resolver)
                 return $this->faker->boolean(90);
             })
             ->setAllowedTypes('enabled', 'bool')
+            ->setDefault('default_tax_zone', LazyOption::randomOneOrNull($this->zoneRepository, 100))
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php]: `LazyOption::randomOneOrNull($this->zoneRepository, 100)` is equivalent to `LazyOption::randomOne($this->zoneRepository)`

**Comment by pjedrzejewski** [comment]
@ktzouno Thanks for the PR, it is good to go once you fix this small thing from review. :)

**Comment by ktzouno** [comment]
Sure. I also had to change the order between channel-grographical in fixtures.yml because `randomOne()` doesn't tolerate an empty result set.

**Comment by pjedrzejewski** [comment]
Thanks @ktzouno, nice work!

**Pull request merged**